src/js/components/DeploymentsListComponent.jsx

```diff
@@ -103,7 +103,7 @@ var DeploymentListComponent = React.createClass({
 
     var headerClassSet = classNames({
       "clickable": true,
-      "dropup": state.sortDescending
+      "dropup": !state.sortDescending
```

Comment:
I really wish there could be a way to see which commit introduced the change in this PR/files view... It would have saved us 2 comments :)